src/endpoints/http/collections/index.js

```diff
@@ -1,9 +1,10 @@
 import listCollections from './list';
 import {createCollection} from './create';
+import authorize from '../../../middleware/route-authorize';
 
 export function collectionsHandler(router) {
   //list collection info
-  router.get('/collections', function(req, res, next) {
+  router.get('/collections', authorize({access: 'read'}), function(req, res, next) {
```

Comment:
to keep things consistent, would it better if we call it "permission", rather than "access"?